src/website/assets/scripts/chi-docs.js

```diff
@@ -108,6 +112,29 @@ onLoad(() => {
     };
   });
 
+  var anchors = document.querySelectorAll('h3,h4');
```

Comment:
The heading tag example on the typography page is not getting excluded from the anchor function. 
![image](https://user-images.githubusercontent.com/130654/57250719-f8e96d80-6ffc-11e9-9b63-4d351e12c320.png)

Reply:
done!